src/hotspot/share/gc/shenandoah/heuristics/shenandoahYoungHeuristics.cpp

```diff
@@ -146,17 +148,6 @@ bool ShenandoahYoungHeuristics::should_start_gc() {
     return true;
   }
 
-  size_t promo_in_place_potential = heap->get_promotion_in_place_potential();
```

Comment:
Can we also now delete `ShenandoahHeap::get_promotion_in_place_potential` and all the attendant code that maintains the value? This heuristic was the only place reading it.

Reply:
Good point.  I'll do that.